### ISSUE TITLE
Env:Ubuntu18.05,ros-melodic,turtlebot3. Runtime error

### ISSUE THREAD

**Issue by CarlDegio** [zerowind168/SIMUROSOT-ROBOCHALLENGE#5]
![prob](https://user-images.githubusercontent.com/52379582/109014830-79f08800-76ef-11eb-94c4-989b8a1a8eca.png)
请问多久可支持在非gazebo7版本上的运行？
How soon this project could run on other gazebo version? or do you have some ideas about this question

**Comment by Jxxxx-2001** [comment]
hello,have you solved this problem?did this project could run on other versions?

**Comment by CarlDegio** [comment]
> hello,have you solved this problem?did this project could run on other versions?

No, may be the world file lead to this question. But the move plugin (.so file) need to be changed if change version. And plugin code is unknown. So it's difficult to change version.

**Comment by Jxxxx-2001** [comment]
> > 你好，你解决了这个问题吗？这个项目可以在其他版本上运行吗？
> 
> 不，可能是世界文件引出了这个问题。但是，如果更改版本，则需要更改移动插件（.so文件）。插件代码是未知的。因此，很难更改版本。
>>ok got it,thank u
